cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -116,9 +125,17 @@ def filtered_list(cls):
         if not switches.is_enabled(u'instructor_paced_only'):
             filtered_list.append('certificate_available_date')
 
-        # Do not show field for configuring Adaptive Learning if feature is disabled.
-        if not AdaptiveLearningEnabledFlag.is_enabled():
-            filtered_list.append('adaptive_learning_configuration')
+        return filtered_list
+
+    @classmethod
+    def course_specific_filtered_list(cls, course_descriptor):
+        """
+        Return list of fields that shouldn't be shown in the Advanced Settings
+        of a given course.
+        """
+        filtered_list = [field_name
+                         for field_name, enabled_function in COURSE_SPECIFIC_FILTERED_LIST
+                         if not enabled_function(course_descriptor)]
```

Comment:
@tomaszgy Formatting is off here. You'll need to do:

```python
        filtered_list = [
            field_name for field_name, enabled_function in cls.COURSE_SPECIFIC_FILTERED_LIST
            if not enabled_function(course_descriptor)
        ]
```

Please review the [EdX Python Style Guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html), in particular the section on [breaking long lines](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines).

Running quality checks locally before pushing changes should also help catch these things.

Reply:
👍 
Thanks! I just ran the quality check and added few more small fixes.